types/resize-observer-browser/test/resize-observer-global.test.ts

```diff
@@ -0,0 +1,11 @@
+function resizeObserverCreatesViaWindow(): void {
+    const resizeObserver = new window.ResizeObserver((entries) => {
+        const div = document.getElementById('display-div')!;
```

Comment:
The name of test fioles is missing `-browser` you want your tests to be named as `resize-observer-browser-global.tests.ts` and `resize-observer-browser-module.tests.ts`

Reply:
Thanks for catching this, fixed.